### ISSUE TITLE
Support relay chain light client

### ISSUE THREAD

**Issue by bkchr** [paritytech/cumulus#439]
Currently we only support full client for the relay chain. We should look into supporting a light client as well.

**Comment by burdges** [comment]
As only a node of the parachain not a collator?

**Comment by bkchr** [comment]
Probably both. However, I would forward this to the parachain developer to ultimately decide on if they may need a full node.

**Comment by burdges** [comment]
We could replicate a subtree between the relay chain and parachain, which helps non-collator parachain full nodes do not require being relay chain full nodes. 

We do simplify several important issues by making collators be full nodes of the relay chain, and even observe realy chain gossip, like they could fetch XCMP messages from approval checkers this way, or discover availability bitfields.

**Comment by bkchr** [comment]
Yeah nothing is decided yet. When we will tackle this, I will make sure to discuss this before ;)
